src/main/resources/reference.conf

```diff
@@ -36,4 +40,6 @@ userprofile {
 
 googlecloud {
   priceListUrl: "https://cloudpricingcalculator.appspot.com/static/data/pricelist.json"
-}
\ No newline at end of file
+}
+
+spray.can.client.response-chunk-aggregation-limit = 5m
```

Comment:
This affects the size of allowable chunks coming from rawls. This number is balanced against the default rawls batch size too. The default is 1m, so bumping this up allows us to grab more content from Rawls in one request.